File: Java/src/main/java/org/sisuns/operadores/basico/Ejercicio_05.java
package org.sisuns.operadores.basico;

import java.util.Scanner;

public class Ejercicio_05 {
    public static void main(String[] args) {
        // Crear un objeto Scanner para leer la entrada del usuario
        Scanner scanner = new Scanner(System.in);

        // Ingresar longitud de la arista
        System.out.print("Ingrese la longitud de la arista: ");
        double arista = scanner.nextDouble();

        // Calcular área
        double area = Math.sqrt(3) * Math.pow(arista, 2);

        // Calcular volumen
        double volumen = (Math.sqrt(2) / 12) * Math.pow(arista, 3);

        // Mostrar resultados
        System.out.println("El área del tetraedro es: " + area);
        System.out.println("El volumen del tetraedro es: " + volumen);

        // Cerrar el objeto Scanner para liberar recursos
        scanner.close();
    }
}

File: PHP/OPERADORES/Basico/Ejercicio_05.php
<?php
    $arista = 5;
    
    //El operador ** es el equivalente a ^ exponente en PHP
    $area = sqrt(3) * $arista ** 2;
    
    $volumen = (sqrt(2) / 12) * $arista ** 3;

    //Mostrar resultados
    echo "El área del tetraedro es: ". round($area, 2)."<br>";
    echo "El volumen del tetraedro es: ". round($volumen, 2)."<br>";
?>